pages/posts/stability.md

```diff
@@ -0,0 +1,137 @@
+the JSON Schema library they are using.
+
+## Our solution
+
+Those are the problems we're aiming to solve in the next release. Instead of
+continuing to release a new immutable and incompatible version of JSON Schema
+with each release, our next release will be a long lived version that is stable,
+but evolving. In this case, "stable" means that there will be strict backward
+and forward compatibility requirements that must be followed for any change. It
+will be much like JavaScript in that as it evolves you can always be sure that
+your existing schemas will continue to work with any implementation you're
+using, but you take a risk using newer features because not all implementations
+will have implemented those features yet.
+
+That vision of a stable yet continuously evolving spec doesn't fit well with the
+IETF process. There are paths we considered, but nothing was proposed that we
+thought would allow us to continue to evolve the standard and get out of "draft"
+any time soon. Therefore, the first step to achieving our vision is to decouple
+from IETF. That allows us to pursue a new model that is more conducive to our
+vision.
+
+Since we want JSON Schema to evolve similarly to JavaScript, we are modeling our
+new process off of the ECMAScript TC39 process. In our next release, most of the
+keywords and features you use today will be declared stable and they will never
+change in a backward incompatible way again. Features and keywords we aren't
+comfortable making stable yet will become part of a new staged release process
+that we are working on defining now. The goal of the staged release process is
+to ensure that the feature gets sufficient implementation, testing, and real
+world vetting for us to be confident in declaring it stable.
+
+## Standards considerations
+
+Starting with the next release, the JSON Schema specification will be
+self-published on our website. One of the concerns about self-publishing is
+about other standards being able to reference the JSON Schema specification. We
+have received feedback from people involved in standards development that our
+approach would be acceptable for them to reference our specification in their
+standards based on our membership with the OpenJS Foundation. We don't know if
+_all_ standards bodies will come to the same conclusion, but this feedback gives
+us confidence that it isn't going to be a significant problem.
+
+## The bottom line
+
+Details about the new process will be shared in separate post when they are
+finalized, but here are some outcomes users can expect.
+
+* If you use only stable features, you are guaranteed interoperability between
+  implementations and you will never need to update those schemas just for the
+  sake of keeping up with new releases.
+* You can use new features before they are stable, but you should only do so if
+  you are in a closed environment where the implementation(s) you will be using
+  support the feature.
```

Comment:
This is too detailed given how little we currently understand about usage patterns.  There may be very strong reasons to rely on "unstable" features in a not-entirely-closed environment.  We don't need to close off this usage before we even know what people need.

Reply:
Good point. I'll try to reword either as an example or in more general terms.